life_expectancy/cleaning.py

```diff
@@ -26,16 +30,32 @@ def clean_data(region_name: str):  # pylint: disable=C0116
     data = data[data['value'] != ' ']
     data = data[data['region'] == region_name]
     data['value'] = data['value'].astype(float)
+    return data
+
+
+def save_data(data) -> csv:  # pylint: disable=C0116
     data.to_csv(DIR_PATH / "data" / SAVE_FILE_NAME, index=False)
 
 
 def parse_args():  # pylint: disable=C0116
+    '''
+    Adding command line option for region.
+    '''
     parser_aux = argparse.ArgumentParser()
     parser_aux.add_argument('--region', type=str, default="PT")
     args = parser_aux.parse_args()
     return args.region
 
 
+def main(region_name):
```

Comment:
for the main function I used the pipe method of pandas, so that you attribute each function as part of the pipeline, based on the way you have your code, you can easily shift into that (or just take a look)